multiqc/modules/hicpro/hicpro.py

```diff
@@ -0,0 +1,386 @@
+             'title': '% Aligned [R1]',
+             'description': 'Percentage of aligned reads [R1] (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+
+         headers['mapped_R2'] = {
+             'title': 'Aligned [R2]',
+             'description' : 'Total number of aligned reads [R2] ({})'.format(config.read_count_desc),
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': True
+         }
+
+         headers['percent_mapped_R2'] = {
+             'title': '% Aligned [R2]',
+             'description': 'Percentage of aligned reads [R2] (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+
+         headers['paired_reads'] = {
+             'title': 'Reported Read Pairs',
+             'description' : 'Total number of reported read pairs ({})'.format(config.read_count_desc),
+             'min' : '0',
+             'scale' : 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key' : 'read_count',
+             'hidden': True
+         }
+
+         headers['percent_paired_reads'] = {
+             'title': '% Reported Read Pairs',
+             'description': 'Percentage of paired reads (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': False
+         }
+
+         headers['valid_interaction'] = {
+             'title': '{} Valid Pairs'.format(config.read_count_prefix),
+             'description': 'Number of valid pairs ({})'.format(config.read_count_desc),
+             'min': 0,
+             'scale': 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key': 'read_count',
+             'hidden': True
+         }
+         
+         headers['percent_valid'] = {
+             'title': '% Valid Pairs',
+             'description': 'Percentage of valid pairs over reported ones (%)',
+             'max': 100,
+             'min': 0,
+             'suffix': '%',
+             'scale': 'YlGn',
+             'hidden': True
+         }
+         
+         headers['valid_interaction_rmdup'] = {
+             'title': '{} Valid Pairs Unique'.format(config.read_count_prefix),
+             'description': 'Number of valid pairs after duplicates removal ({})'.format(config.read_count_desc),
+             'min': 0,
+             'scale': 'RdYlBu',
+             'modify': lambda x: x * config.read_count_multiplier,
+             'shared_key': 'read_count',
+             'hidden': True
+         }
+         
+         headers['percent_valid_interaction_rmdup'] = {
+             'title': '% Valid Pairs Unique',
```

Comment:
When I've had similar data in the past, I've inverted it. I think that _% duplicate_ is easier to understand than _% unique_ personally (especially if it's covered in reds / oranges to show that a big number is bad). As it stands, I was confused at first as to how this value could be higher than _% Reported read pairs_ for sample `mF1216` as I (wrongly) assumed that it was also a percentage of the total read count.

If you want to do this, it's pretty easy. Just add the following line to the config to modify the data on the fly for the table:

```python
'modify': lambda x: 100.0 - x
```

Reply:
I agree but here the 'valid_interaction_rmdup' is the final number of reported reads (after duplicates removal). If I replace by number of duplicates, I'm afraid people will not really see how many reads are kept at the end

Reply:
Sure, but I'm referring only to the percentage column here, not the remaining read count.

Reply:
ok. Will change replace by the % of duplicates